### PR TITLE
174 | Improving installation instructions

### LINKED ISSUE #174: Improve installation instructions
User feedback suggests the installation instructions could be improved and clarified.

* Python >= 3.8 required.
* Setting up a new/clean Conda environment.

### PR DESCRIPTION
**Pull request by ns-rse**
Resolves #174

### CODE REVIEW

**Review by SylviaWhittle — APPROVED** [review]
This is a really useful addition to the install instructions, thank you @ns-rse

File: docs/installation.md [hunk 1]
```diff
@@ -3,10 +3,32 @@
...
 workin on the the `dev` branch.
 
-Currently TopoStats should be installed from the git repository, ideally under a [Python Virtual
-Environment](https://realpython.com/python-virtual-environments-a-primer/)
-(e.g. [Miniconda](https://docs.conda.io/en/latest/miniconda.html)).
+TopoStats is a [Python](https://www.python.org) package designed to run at the command line. You may have Python
+installed on your system but should use a [Python Virtual
+Environment](https://realpython.com/python-virtual-environments-a-primer/) such as
+[Miniconda](https://docs.conda.io/en/latest/miniconda.html) and install it under the Virtual Environment. The versions
+of Python supported is Python >=3.8 and so when creating your virtual environment you should specify this `3.8` as the
```
Comment (bobturneruk) [inline docs/installation.md:10]:
```suggestion
of Python supported are Python >=3.8 and so when creating your virtual environment you should specify this `3.8` as the
```

File: docs/installation.md [hunk 2]
```diff
@@ -3,10 +3,32 @@
...
+`python=3.10`). After creating it we can, as per the instructions printed out, activate the environment.
+
+``` bash
+conda create --name topostats python=3.10
+conda activate topostats
+```
+
+You are now ready to clone and install TopoStats.
+
+## Cloning and Installing
```
Comment (bobturneruk) [inline docs/installation.md:27]: This is an option that doesn't need `git`:

```
pip install git+https://github.com/AFM-SPM/topostats@dev
```

However, on reflection maybe not the best one as it loses the version hash which we might need to help fix user problems (until we switch to semantic versioning, which is captured by `pip`).

File: docs/installation.md [hunk 3]
```diff
@@ -3,10 +3,32 @@
...
+
+Currently TopoStats has not been released to the [PyPi](https://pypi.org) repository (although a release is planned). In
+the mean time you have to clone the repository from GitHub and install it. Work is on-going on the `dev` branch and so
+to test that you need to "checkout" this branch prior to installing it.
 
 ``` bash
 git clone git@github.com:AFM-SPM/TopoStats.git
```
Comment (bobturneruk) [inline docs/installation.md:34]:
```suggestion
git clone -b dev https://github.com/AFM-SPM/TopoStats.git
```

Then don't need to checkout.

File: docs/installation.md [hunk 4]
```diff
@@ -3,10 +3,32 @@
...
+conda create --name topostats python=3.10
+conda activate topostats
+```
+
+You are now ready to clone and install TopoStats.
+
+## Cloning and Installing
+
+Currently TopoStats has not been released to the [PyPi](https://pypi.org) repository (although a release is planned). In
+the mean time you have to clone the repository from GitHub and install it. Work is on-going on the `dev` branch and so
```
Comment (bobturneruk) [inline docs/installation.md:30]: Please include a link to instructions on how to install `git`.

**Review by bobturneruk — APPROVED** [review]
Thanks for this. A good improvement. As noted, changes to make this easier still are in the pipeline.

File: docs/installation.md (hunk 4 above)
Comment (ns-rse) [reply docs/installation.md]: Done :+1:

**Pull request merged**